package.json

```diff
@@ -1,13 +1,14 @@
 {
     "name": "probe-rs-debugger",
     "displayName": "Debugger for probe-rs",
-    "version": "0.17.0",
+    "version": "0.17.1",
```

Comment:
Is the idea here to track the probe-rs version?

Reply:
That is what @Yatekii suggested, and I'm OK to track 'major.minor', but necessity might make it difficult to keep the '.patch' in synch. VSCode marketplace won't allow you to publish updates to the extension unless you  bump at least the patch.

Reply:
I'm a bit worried that it's almost more confusing if it's almost the same, since probe-rs itself might also get patch release.

Reply:
I honestly don't think it will be an issue. If you have a counter proposal that @Yatekii will align to, then I'm open to adjusting.

Reply:
I'm fine with it, just my 2 cents.